src/java.base/share/classes/sun/launcher/LauncherHelper.java

```diff
@@ -57,26 +57,22 @@
 import java.nio.file.DirectoryStream;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.security.NoSuchAlgorithmException;
+import java.security.Provider;
+import java.security.Security;
 import java.text.MessageFormat;
 import java.text.Normalizer;
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.Comparator;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Locale;
+import java.util.*;
```

Comment:
Wild card imports are discouraged. (Your IDE may have done this behind your back)